moderatorFrontend/src/main/kotlin/views/common/GenericErrorView.kt

```diff
@@ -43,51 +54,51 @@ class PathNotFound : RComponent<GenericErrorViewProps, RState>() {
   }
 }
 
-class GenericErrorView : RComponent<GenericErrorViewProps, RState>() {
-  override fun RBuilder.render() {
-    div(props.classes.centeredDiv) {
-      typography {
-        attrs.className = props.classes.centeredText
-        attrs.variant = "h5"
+private class GenericErrorView : RComponent<GenericErrorViewProps, State>() {
+  override fun ChildrenBuilder.render() {
+    Box {
+      sx = jso {
+        margin = Auto.auto
+      }
+      Typography {
+        sx {
+          centeredText()
+        }
```

Comment:
When do we use `js = jso { ... }` and when `js { ... }`?

Reply:
Very nice finding, thanks!
`sx = jso { ... }` is already kind of legacy style because the new `PropsWithSx.sx` (`sx { ... }`) has the `PropertiesBuilder` block which makes it possible to access Selectors like `hover`, select classes etc.
I already unified these two usages now :)